.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this added by accident? Likely bad `.gitignore` for this folder. Always review your own PR please.